tests/phpunit.xml

```diff
@@ -1,22 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.5/phpunit.xsd"
     cacheResult="true"
     beStrictAboutOutputDuringTests="true"
     backupGlobals="false"
     backupStaticAttributes="false"
     bootstrap="phpunit-bootstrap.php"
     colors="true"
     convertErrorsToExceptions="true"
+    testdox="true"
     convertNoticesToExceptions="true"
     convertWarningsToExceptions="true"
     processIsolation="false"
-    stopOnFailure="true"
-    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+    stopOnFailure="true">
     <testsuites>
-        <testsuite name="Common Test Suite">
+        <testsuite name="LibraryClasses">
             <directory suffix="Test.php">classes</directory>
         </testsuite>
+        <testsuite name="ApplicationClasses">
```

Comment:
:+1:, much better to be using the PHPUnit suite management rather than reinventing it in shell scripts!